### PR TITLE
compute_val_loss parser action to store_true

### PR DESCRIPTION
**Pull request by mrtgenet**
Either the argument should be named "--no-val-loss" or something like this and have a store_false action, or be named "--compute-val-loss" and have a store_true action. Second option is more coherent with the fact that providing a validation set is optional.

See [this issue](https://github.com/fizyr/keras-retinanet/issues/976)

### CODE REVIEW

**Comment by de-vri-es** [comment]
Looks good. Once CI passes, we can merge. Thanks for the PR!

**Comment by de-vri-es** [comment]
It passed! \o/
